cmd/git-sync/main.go

```diff
@@ -58,7 +58,7 @@ var flSyncTimeout = flag.Int("timeout", envInt("GIT_SYNC_TIMEOUT", 120),
 var flOneTime = flag.Bool("one-time", envBool("GIT_SYNC_ONE_TIME", false),
 	"exit after the initial checkout")
 var flMaxSyncFailures = flag.Int("max-sync-failures", envInt("GIT_SYNC_MAX_SYNC_FAILURES", 0),
-	"the number of consecutive failures allowed before aborting (the first pull must succeed)")
+	"the number of consecutive failures allowed before aborting (the first pull must succeed, <0 allows for infinite failures)")
```

Comment:
Would be specific here. No need to allow -1+. `"-1 disables aborting for any number of failures"`

Reply:
Is your suggestion just to doc it as such? Or to actually change the code to only do this if `-1`?

Reply:
So doc change for sure as there is no need to make it more complicate as needed.
On enforcing this. I think being explicit is a better long term thing even, if it makes the code a bit longer, but it explicitly "documents" this within the code. So my vote for also enforcing it.